test/integration/patients/patient/patient-flow.js

```diff
@@ -141,6 +142,7 @@ context('patient flow page', function() {
         'form': getRelationship(testForm),
         'patient': getRelationship(testPatient),
         'program-action': getRelationship(testProgramAction),
+        'files': getRelationship([{ id: testOtherFileId }], 'files'),
```

Comment:
Adding an existing file for the action. To ensure that it isn't removed when we set the new `_files` relationship.